### PR TITLE
Intercept callbacks for entry change / create

### PR DESCRIPTION
**Pull request by tnagorra**
Merge https://github.com/toggle-corp/react-store/pull/362 for great results

### CODE REVIEW

File: src/redux/reducers/siloDomainData/editEntries.js [hunk 1]
```diff
@@ -563,7 +570,10 @@ const applyToAllEntries = mode => (state, action) => {
     iterableEntries.forEach((entry, i) => {
         if (entry === undefined) {
             return;
+        } else if (!modifiable && !!entryAccessor.serverId(entry)) {
+            return;
         }
+
```
Comment (AdityaKhatri) [inline src/redux/reducers/siloDomainData/editEntries.js:576]: line farming

File: src/views/EditEntries/index.js [hunk 2]
```diff
@@ -295,21 +299,42 @@ export default class EditEntries extends React.PureComponent {
...
 
-    setEntryData = (val) => {
-        console.warn('Changing entry');
-        this.props.setEntryData(val);
-    }
-
     setExcerpt = (val) => {
-        console.warn('Changing excerpt');
+        if (this.shouldDisableEntryChange(val.id)) {
+            console.warn('No permission to edit entry excerpt');
```
Comment (AdityaKhatri) [inline src/views/EditEntries/index.js:304]: Shouldn't you notify the user instead of the browser

File: src/views/EditEntries/index.js [hunk 3]
```diff
@@ -295,21 +299,42 @@ export default class EditEntries extends React.PureComponent {
...
+            return;
+        }
         this.props.setExcerpt(val);
     }
 
+    setEntryData = (val) => {
+        if (this.shouldDisableEntryChange(val.id)) {
+            console.warn('No permission to edit entry');
```
Comment (AdityaKhatri) [inline src/views/EditEntries/index.js:312]: Shouldn't you notify the user instead of the browser

File: src/views/EditEntries/index.js [hunk 4]
```diff
@@ -295,21 +299,42 @@ export default class EditEntries extends React.PureComponent {
...
+            return;
+        }
+        this.props.setEntryData(val);
+    }
+
     addEntry = (val) => {
-        console.warn('Adding entry');
+        if (this.shouldDisableEntryCreate()) {
+            console.warn('No permission to create entry');
```
Comment (AdityaKhatri) [inline src/views/EditEntries/index.js:320]: Shouldn't you notify the user instead of the browser

File: src/views/EditEntries/index.js (hunk 2 above)
Comment (tnagorra) [reply src/views/EditEntries/index.js:304]: notify garina, harek letter type garda euta notification aucha natra

File: src/views/EditEntries/index.js (hunk 3 above)
Comment (tnagorra) [reply src/views/EditEntries/index.js:312]: notify garina, harek letter type garda euta notification aucha natra

File: src/views/EditEntries/index.js (hunk 4 above)
Comment (tnagorra) [reply src/views/EditEntries/index.js:320]: notify garina, harek letter type garda euta notification aucha natra

File: src/redux/reducers/siloDomainData/editEntries.js (hunk 1 above)
Comment (tnagorra) [reply src/redux/reducers/siloDomainData/editEntries.js:576]: line farming haina, chaine thauna ma ta line rakhna paryo ni
block chuttaideko ho

File: src/views/EditEntries/index.js (hunk 2 above)
Comment (AdityaKhatri) [reply src/views/EditEntries/index.js:304]: Ramro ho ni tyo tah

File: src/views/EditEntries/index.js (hunk 3 above)
Comment (AdityaKhatri) [reply src/views/EditEntries/index.js:312]: Ramro ho ni tyo tah

File: src/views/EditEntries/index.js (hunk 4 above)
Comment (AdityaKhatri) [reply src/views/EditEntries/index.js:320]: Ramro ho ni tyo tah

**Pull request merged**